.eslintrc.js

```diff
@@ -336,6 +337,9 @@ module.exports = {
     'use-isnan': 'error',
     'valid-typeof': ['error', { requireStringLiterals: true }],
 
+    // JSDoc rules
+    'jsdoc/check-types': 'error',
```

Comment:
Is this the only rule we want or is better to go with `"extends": ["plugin:jsdoc/recommended"` and then turn off the ones that you don't?

Reply:
I was going to enable them one by one but your idea here is probably better.

Reply:
(I might land this as-is and then do your suggestion in a subsequent PR.)

Reply:
Just noticed that it doesn't look like `extends` is used at all in this repo, but you can also inline the rules marked as recommended from https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-configuration. You're right that that could definitely be separate